discord_commands/drop_simulations.py

```diff
@@ -74,5 +74,7 @@ async def cmd_dry(ctx, name: str, trials = 100, extraArg1 = "True", extraArg2 =
     description = probability_flavour_text.get_flavour(probability)
     embed = discord.Embed(title=title, description=description)
     if extraArg1 == "True" and name != "tob":
-        embed.set_footer(text="Ignored drops: " + str(trash_drops))
+        last_trash_drop = trash_drops.pop()
```

Comment:
Instead of popping, use negative indecies to get entries from the end of the collection, rather than the start (e.g, trash_drops[-1]).